OpenRA.Mods.Common/UtilityCommands/UnpackMapCommand.cs

```diff
@@ -0,0 +1,65 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.IO;
+using System.Linq;
+using OpenRA.FileSystem;
+
+namespace OpenRA.Mods.Common.UtilityCommands
+{
+	sealed class UnpackMapCommand : IUtilityCommand
+	{
+		string IUtilityCommand.Name => "--unpack-map";
```

Comment:
IMO having this command act on all maps isn't particularly useful outside of cases like this PR, but having it target an individual map would be much more useful. Also note that we have an existing `--refresh-map` command.

What do you think about either:
* Have separate `--unpack-map`, `--pack-map` commands that act on a given file/folder path (and keep `--refresh-map` as-is).
* Have a single `--map (pack|unpack|refresh)` command that acts on a given file/folder path.

?

Reply:
Added an extra commit to implement the latter suggestion with a regex filter for filenames to operate on.